genie/database_to_staging.py

```diff
@@ -498,6 +498,19 @@ def seq_date_filter(clinicalDf, processingDate, consortiumReleaseCutOff):
     return removeSeqDateSamples
 
 
+def sample_class_filter(clinical_df: pd.DataFrame) -> list:
+    """Filter samples by SAMPLE_CLASS
+
+    Args:
+        clinical_df (pd.DataFrame): Clinical dataframe
+
+    Returns:
+        list: List of samples to filter out
+    """
+    remove_samples = clinical_df["SAMPLE_ID"][clinical_df["SAMPLE_CLASS"] == "cfDNA"]
```

Comment:
Do we know that "cfDNA" is the SAMPLE_CLASS value?

Reply:
There is only "Tumor" and "cfDNA" as allowed values here, so I'm assuming we don't want to filter out the 'Tumor' values?